src/helpers/date-formatter.helper.test.ts

```diff
@@ -0,0 +1,17 @@
+import { salesforceFormattedCurrentDate } from './date-formatter.helper';
+
+describe('salesforceFormattedCurrentDate function', () => {
+  const mockDate = new Date(2007, 3, 27);
+
+  beforeEach(() => {
+    jest.spyOn(global, 'Date').mockImplementation(() => mockDate);
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('should return the current date, formatted correctly', () => {
```

Comment:
Can we add more test cases:

1. Past date 1970-01-01.
2. Future date 9999-99-99.
3. Leap year date 2020-02-29.